libnavigation-base/build.gradle

```diff
@@ -33,6 +33,7 @@ dependencies {
     api dependenciesList.mapboxCore
     api dependenciesList.mapboxSdkGeoJSON
     api dependenciesList.mapboxSdkDirectionsModels
+    implementation dependenciesList.mapboxSdkTurf
```

Comment:
Do we need this dependendency here?

Reply:
needed for `ReplayRouteLocationConverter.kt`. Moved it to core. Removed dependency